monitor_mqtt/actions/clean_data_mqtt_ingest_chain.meta.yaml

```diff
@@ -0,0 +1,16 @@
+---
+  name: "clean_data_mqtt_ingest_chain"
+  description: "Action Chain to pre process CSV metrics data and ingest it to monitor via mqtt."
```

Comment:
Just kind of curious of how one would use this action?
From my reading, you would write some data to a file on the filesystem, wherever you have `data_file_path` in your pack configuration, then the `clean_data` action would read the data from the file, clean it, write it back out to the file `/opt/stackstorm/packs/monitor_mqtt/etc/clean_data_output/clean_data.csv`. Next `publishcsvdata` would try to read data from the `csvDataFile` in your pack config and send it to the MQTT topic.

Seems like either `clean_data` should use `csvDataFile` config option, or `publishcsvdata` should use the static path. Or, the path of data written should be output from `clean_data` and passed in to `publishcsvdata`?

Reply:
clean_data Action flow   
I - data_file_path , json_schema_path (from Config file)    O - /etc/clean_data_output/clean_data.csv

publishcsvdata Action flow  
I - csvFilePath(from Config file) to publish CSV data to MQTT

For Action chain to work properly :
csvFilePath(from Config file) must have same path of output file generated by  clean_data Action i.e. 
(static path  inside packs location: /etc/clean_data_output/clean_data.csv)